blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -286,12 +294,15 @@ export class RunDetailsPipeline extends Component {
                 pathArray.shift();
                 newPath = `${pathArray.join('/')}/`;
             }
+            const isParallel = this.mergedConfig.nodeReducer.edges[0] && nodeInfo.edges[0] ?
+                this.mergedConfig.nodeReducer.edges[0].id === nodeInfo.edges[0].id : false;
```

Comment:
I read a comment on one other other PRs to the effect that "less code is always better". Naive, to say the least. 

As someone else rightly said (can't remember who atm) ... "Any clown can write code that a computer can understand, but it takes a good programmer to write code that a human can understand". Imo, less code is not always better. The fewest number of easily understood/comprehended lines of code ... sure, that's good.